docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/SaveAndLoadFromText.cs

```diff
@@ -0,0 +1,98 @@
+            var mlContext = new MLContext(seed: 0);
+
+            // Create a list of training data points.
+            IEnumerable<DataPoint> dataPoints = GenerateRandomDataPoints(10);
+
+            // Convert the list of data points to an IDataView object, which is consumable by ML.NET API.
+            IDataView data = mlContext.Data.LoadFromEnumerable(dataPoints);
+
+            // Inspect the data before saving to a binary file.
+            PrintPreviewRows(dataPoints);
+
+            // The rows in the data.
+            // 0, 0.7262433
+            // 1, 0.8173254
+            // 0, 0.7680227
+            // 1, 0.5581612
+            // 0, 0.2060332
+            // 1, 0.5588848
+            // 0, 0.9060271
+            // 1, 0.4421779
+            // 0, 0.9775497
+            // 1, 0.2737045
+
+            // Create a FileStream object and write the IDataView to it as a binary IDV file. 
+            using (FileStream stream = new FileStream("data.tsv", FileMode.Create))
+            {
+                mlContext.Data.SaveAsText(data, stream);
+            }
+
+            // Create an IDataView object by loading the binary IDV file.
+            IDataView loadedData = mlContext.Data.LoadFromTextFile("data.tsv");
+
+            // Inspect the data that is loaded from the previously saved binary file.
+            var loadedDataEnumerable = mlContext.Data.CreateEnumerable<DataPoint>(loadedData, reuseRowObject: false);
+            PrintPreviewRows(loadedDataEnumerable);
+
+            // The rows in the data.
+            // 0, 0.7262433
+            // 1, 0.8173254
+            // 0, 0.7680227
+            // 1, 0.5581612
+            // 0, 0.2060332
+            // 1, 0.5588848
+            // 0, 0.9060271
+            // 1, 0.4421779
+            // 0, 0.9775497
+            // 1, 0.2737045
+
+            File.Delete("data.tsv");
+        }
+
+        private static IEnumerable<DataPoint> GenerateRandomDataPoints(int count, int seed = 0)
+        {
+            var random = new Random(seed);
+            for (int i = 0; i < count; i++)
+            {
+                yield return new DataPoint
+                {
+                    Label = i % 2,
+
+                    // Create random features that are correlated with label.
+                    Features = (float)random.NextDouble()
+                };
+            }
+        }
+
+        // Example with label and feature values. A data set is a collection of such examples.
+        private class DataPoint
+        {
+            public float Label { get; set; }
+
+            public float Features { get; set; }
+        }
+
+        // Print helper.
+        private static void PrintPreviewRows(IEnumerable<DataPoint> data)
+        {
+            Console.WriteLine($"The rows in the data.");
+            foreach (var row in data)
+                Console.WriteLine($"{row.Label}, {row.Features}");
```

Comment:
>Console.WriteLine($"{row.Label}, {row.Features}"); [](start = 16, length = 50)

you can move this to ToString() definition for DataPoint class and get rid of PrintPreviewRows function altogether. You only need to preview once you read back the data. you could just do the following inside Example() w/o any extra function:
foreach d in data
    WriteLine(d);